docs/cloud/data-stores/duarability-vs-high-availability.md

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 3
+title: Durability and Availability
+--- 
+
+
+# Durability and Availability
+Dragonfly Cloud prioritizes both data durability and high availability for your data stores. You can further fine-tune these aspects through the concept of eviction policies when creating a data store. This allows you to strike a balance between data persistence and performance based on your application's needs.
```

Comment:
please stick to the content I specified in
https://docs.google.com/document/d/1kipawV9069mc8iSZEP6OuiVsGlQkS6JTs5eZEQAYjMo/edit

Reply:
make sure to do so across the entire PR